indico/modules/events/registration/lists.py

```diff
@@ -145,8 +145,7 @@ def _get_sorted_regform_items(self, item_ids):
     def _get_filters_from_request(self):
         filters = super()._get_filters_from_request()
         for field in self.regform.form_items:
-            if field.is_field and field.input_type in {'single_choice', 'multi_choice', 'country', 'bool', 'checkbox',
-                                                       'sessions'}:
+            if field.is_field and hasattr(field, 'field_impl') and field.field_impl.filter_choices:
```

Comment:
`hasattr(field, 'field_impl')` is unnecessary. This attribute always exists in case of `is_field`.

```suggestion
            if field.is_field and field.field_impl.filter_choices:
```

Reply:
Your reaction time is uncanny :)

In my original version I made the check before is_field and ran into missin field_impl. Good to know that this is covered by is_field. Makes it much more readable.